tests/shotgun_model/test_data_handler_find.py

```diff
@@ -171,26 +181,29 @@ def test_updates(self):
         # we are getting a diff of two added nodes back.
         self.assertEqual(len(diff), 4)
 
+        # Items will be sorted as /Character, /Prop, 1234 and 3333.
```

Comment:
Do a deterministic sort of the entries so Python 3's output is predictable.

Reply:
👍